build.gradle

```diff
@@ -818,7 +819,10 @@ project('test:system') {
         if (System.getProperty("imageVersion") == null) {
             systemProperty "imageVersion", "INVALID_IMAGE_VERSION"
         }
-
+        
+        systemProperty "securityEnabled", System.getProperty("securityEnabled", "false")
+        println "Is security enabled for tests:" + System.getProperty("securityEnabled")
```

Comment:
The print line will printed for all for all the gradle tasks, this is because this line will be executed during configuration phase of gradle's build lifecycle.
We should remove it or associate it with a gradle action...

Reply:
Sure will remove this.